### PR TITLE
Full Outline

### PR DESCRIPTION
**Pull request by Ampersandstorm**
This PR is branched from PR #6 and adds only the completed outline (the SUMMARY.md file). Please review and let me know if there are any suggestions or concerns.

### CODE REVIEW

**Review by jacobgkau — COMMENTED** [review]
I can imagine a few things potentially being rearranged as the content is developed, but I think this outline is a good comprehensive target to work towards. I just had a few small notes.

File: src/SUMMARY.md [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
-    
-- [Navigate Pop!_OS]()
-- [Customize Pop!_OS]()
+        - [First-time Setup]()
+            - [Initial Setup Steps]() <!--First-time setup windows-->
+            - [Connecting to Wi-Fi]()
+            - [Privacy Settings]()
+[Navigate Pop!\_OS](Navigate-Pop/navigate-pop.md)
+    - [COSMIC & Pop!\_Shell]() <!--Define and describe pop shell and cosmic environment-->
+    - [Tiling & Stacking Windows]()
```
[the remark this answers is not in the document]
Comment (Ampersandstorm) [reply src/SUMMARY.md:21]: After digging more into COSMIC, I'm not sure where I want to include this. If I understand correctly, COSMIC components are also present in initial setup and content that will be in the "Customize Pop!_OS" chapters. I'm starting to think this should go in the "What is Pop!_OS?" section.

[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/SUMMARY.md:21]: That might make sense. COSMIC is the overarching interface project encompassing a lot of the components involved in day-to-day use of the system, so a COSMIC page is going to be very general and introductory.

**Comment by Ampersandstorm** [comment]
Converting this to a draft because PRs #6 and #9 introduce changes to the underlying folder structure for this mdbook. I want to get those PR's approved before tackling the many merge conflicts this PR will introduce.

**Comment by jackpot51** [comment]
This will have to be rebased, and then reopened.

**Pull request closed without merging**